charts/alchemist/README.md

```diff
@@ -45,4 +45,4 @@ Transforms data from the raw topic and writes it to transformed topic
 | tolerations | list | `[]` |  |
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.12.0](https://github.com/norwoodj/helm-docs/releases/v1.12.0)
+Autogenerated from chart metadata using [helm-docs v1.11.0](https://github.com/norwoodj/helm-docs/releases/v1.11.0)
```

Comment:
It seems weird that the helm docs version changed from 1.12 -> 1.11. Was that intentional?

Reply:
Fixed, upgraded to 1.13.1, so that should be the new autogenerated version